src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -288,9 +288,8 @@ that coordinates are expected in cartesian coordinates according to the
 `*_coordsystem.json` file MUST be specified as well.
 
 The optional [`space-<label>`](../99-appendices/09-entities.md#space) entity (`*[_space-<label>]_electrodes.tsv`) can be used to
-indicate the way in which electrode positions are interpreted. The space label
-needs to be taken from the list in
-[Appendix VIII](../99-appendices/08-coordinate-systems.md)
+indicate the way in which electrode positions are interpreted.
+The space `<label>` MUST be taken from the list in [Appendix VIII](../99-appendices/08-coordinate-systems.md)
```

Comment:
Does this mean any identifier, or just the standard identifiers?

Reply:
very good point: this is way too unclear. What is meant is: if `space` is used in iEEG data, the iEEG keywords apply ... if with EEG, the EEG ..... and so on

Reply:
how about this?

```suggestion
The space `<label>` MUST be taken from one of the modality specific lists in
[Appendix VIII](../99-appendices/08-coordinate-systems.md).
For example for iEEG data, the restricted keywords listed under
[iEEG Specific Coordinate Systems](../99-appendices/08-coordinate-systems.md#ieeg-specific-coordinate-systems)
are acceptable for `<label>`.
```

Reply:
Okay sorry maybe I misunderstood along the way. 

Shouldn't any of the standard template identifiers be okay as well? And "Other"

E.g. space-fsaverage

Reply:
Yes, but the ieeg list includes the names like fsaverage and so on

Reply:
see the last bullet point

![image](https://user-images.githubusercontent.com/9084751/109414799-1e572080-79b5-11eb-88a7-e2d0bcd87870.png)

screenshot from: https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html#ieeg-specific-coordinate-systems

Reply:
Ah right you are.